### ISSUE TITLE
[Accessibility]: hljs is coloring Copilot's code

### ISSUE THREAD

**Issue by yanskun** [sourcegraph/sourcegraph#61528]
### Problem description

When I install the Chrome extension, the source code of Copilot's Chat Panel is missing and the code block is not readable because the same color as the background-color is applied.

### Expected behavior

Restricted from working inside Copilot's Chat Panel

### Additional details

**current**
![image](https://github.com/sourcegraph/sourcegraph/assets/43776161/a2486eff-2824-4afd-93ed-146c15fe659a)
![image](https://github.com/sourcegraph/sourcegraph/assets/43776161/9afc3934-1161-4a1a-b178-5c6702d6d586)

Maybe this style
https://github.com/sourcegraph/sourcegraph/blob/849c7e04b9eb4f7341d5fd5db7aeb656dacbdedc/client/browser/src/highlight.scss#L6-L11
